.drone.yml

```diff
@@ -0,0 +1,104 @@
+workspace:
+  base: /drone
+  path: src
```

Comment:
set the `path` to `server/apps/files_primary_s3`, than you can sync the core to `/drone/server` and you got everything in place.

Reply:
I doubt that I can call git clone into an existing folder? or am I missing anything? THX

Reply:
true, that have to happen before. Or requires something like rsync/cp -rf

Reply:
or can we customize the clone step?

Reply:
It's possible to clone something else first, this is just written out of my head so it needs to be confirmed that it works as intended.

```
workspace:
  base: /drone
  path: server/apps/files_primary_s3

clone:
  core:
    image: plugins/git
    remote: https://github.com/owncloud/core.git
    path: /drone/server
    ref: refs/heads/master
    recursive: true

  git:
    image: plugins/git

pipeline:
  restore:
    image: plugins/s3-cache:1
    pull: true
    secrets: [ cache_s3_server, cache_s3_access_key, cache_s3_secret_key ]
    restore: true
    when:
      local: false
```

This changed `clone` step SHOULD clone the core into `/drone/server` and afterwards the app into `/drone/server/apps/files_primary_s3`. Attributes like the `ref` can be replaced by matrix variables later on to run the tests against different core versions.